public/style/components/_data.scss

```diff
@@ -0,0 +1,10 @@
+.data-primary {
```

Comment:
`_data.scss` and _`forecast.scss` are so similar in scope that I wonder if they could be merged. I'm not sure what the best name would be. `data` stuff is really just changing the style for forecast data.